Templates/Resources/Templates/___FILEBASENAME___Interfaces.swift.erb

```diff
@@ -0,0 +1,63 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//  Copyright (c) ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  This file was generated by the 🐍 VIPER generator
+//
+
+import UIKit
+<% if @template.generate_io -%>
+import RxSwift
+import RxCocoa
+<% end -%>
+
+protocol ___VARIABLE_moduleName___WireframeInterface: WireframeInterface {
+}
+
+protocol ___VARIABLE_moduleName___ViewInterface: ViewInterface {
+}
+
+protocol ___VARIABLE_moduleName___PresenterInterface: PresenterInterface {
+<% if @template.generate_io -%>
+    func configure(with output: ___VARIABLE_moduleName___.ViewOutput) -> ___VARIABLE_moduleName___.ViewInput
+<% end -%>
+}
+
+<% if @complexity.generate_formatter -%>
+protocol ___VARIABLE_moduleName___FormatterInterface: FormatterInterface {
+    <%- if @template.generate_io -%>
+    func format(for input: ___VARIABLE_moduleName__.FormatterInput) -> ___VARIABLE_moduleName__.FormatterOutput
```

Comment:
Missing one `_` at the end of both `___VARIABLE_moduleName__`

Reply:
👍